lsp-headerline.el

```diff
@@ -307,7 +307,7 @@ PATH is the current folder to be checked."
   (let ((range-severity 10))
     (mapc (-lambda ((&Diagnostic :range (&Range :start) :severity?))
             (when (lsp-point-in-range? start range)
-              (setq range-severity (min range-severity severity?))))
+              (setq range-severity (min range-severity (or severity? 10)))))
```

Comment:
`severity?` can be nil which `min` fn will not accept

Reply:
How about adding the `(and severity?` in the `when` clause? 🤔 I'm trying to avoid the magic number and improve the readability. :)

Reply:
yeah it's better your way, updated.